django_project/ta_system/templates/ta_system/home.html

```diff
@@ -1,4 +1,19 @@
 {% extends "ta_system/base.html" %}
 {% block content %}
-    <h1>Home Page</h1>
-{% endblock content %}
\ No newline at end of file
+    <div class="content-section">
+        {% if last_system_status.status %}
+            <h3> The TA Application System is Open. Apply now!</h3>
+        {% elif last_system_status.date_changed %}
+            <h3> The TA Application System is Closed at {{last_system_status.date_changed}}.</h3>
```

Comment:
Did Prof. Samary say we needed to include the date here? I feel like there's not much use to the students knowing when it was opened, and leaving it out avoids that awkward problem where there is no status until the admin changes the status for the first time. I can see how it would be useful for debugging, but let me know what you think.

Reply:
The default status of the system is closed, so even if the admin has not changed the status, it will still show the system is closed (see line 8-9). 
I removed the date changed from the home page as it will be easier to pass and display the variable "system_is_open" as the status of the instance rather than the whole instance.